cmdline-pull/pubsub_sample.py

```diff
@@ -105,6 +108,30 @@ def create_topic(client, args):
     print 'Topic {} was created.'.format(topic['name'])
 
 
+def grant_gmail_push(client, args):
+    """Grants gmail push notification on this topic."""
+    topic = get_full_topic_name(args.project_name, args.topic)
```

Comment:
Hmm, as nice is it is to do that, I would prefer main to explicitly pass the args. Zen of Python sort of thing.